jobs/config.json

```diff
@@ -12502,6 +12503,7 @@
       "--gcp-zone=us-east1-a",
       "--provider=gce",
       "--stage=gs://kubernetes-release-pull/ci/pull-kubernetes-e2e-gce-big-performance",
+      "--tear-down-previous",
```

Comment:
This is not the job... You need to change it for kubemark-big.

Reply:
ah.. good catch